src/main/webapp/app/exercises/shared/feedback/item/feedback-item-service.ts

```diff
@@ -29,14 +29,41 @@ export class FeedbackItemServiceImpl implements FeedbackItemService {
 
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     create(feedbacks: Feedback[], showTestDetails: boolean): FeedbackItem[] {
-        return feedbacks.map((feedback) => ({
+        return feedbacks.map((feedback) => this.createFeedbackItem(feedback, showTestDetails));
+    }
+
+    createFeedbackItem(feedback: Feedback, showTestDetails: boolean): FeedbackItem {
```

Comment:
This is a quickfix for now and we plan to move more of this logic towards the server so that parsing from `Feedback -> FeedbackItem` is no longer necessary